### PR TITLE
O-la-la docs

### PR DESCRIPTION
**Pull request by qfox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,105 @@
...
+```
+
+### `BundleBuilder~Bundle`
+
+[`BemBundle`][] class extended by `src` and `target` helper methods.
+
+### `BundleBuilder~Bundle.src`
+
+Gather vinyl file objects that responde to [BEM entity][] declaration
+and their dependencies and returns them as a readable stream.
```
Comment (tadatuta) [inline README.md]: return~~s~~

**Comment by tadatuta** [comment]
👍

**Comment by skad0** [comment]
👍💥

**Comment by qfox** [comment]
@blond @skad0 ty, fellows!

**Pull request merged**